styleguide/source/assets/scss/02-molecules/_inline-bio.scss

```diff
@@ -3,11 +3,11 @@
   padding: 20px 0;
   border-top: 1px solid $gray-50;
   border-bottom: 1px solid $gray-50;
-  display: grid;
-  grid-template-columns: 1fr 8fr;
```

Comment:
You probably wants something a little more static here such as 
grid-template-columns: auto 1fr;

or better yet
grid-template-columns: 115px 1fr;

Reply:
Also @designerneil I do not think that CSS grid is the right approach here for the reason that created this bug. I think the correct approach here is to use `display: inline-block;` when appropriate (since this is the direct use case for that). 

Please see how I addressed the same problem with the Information Card pattern - https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/develop/styleguide/source/assets/scss/02-molecules/_info-card.scss

Reply:
@labbydev -- I agree. Should I remove CSS grid approach?